src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.nocpm.props

```diff
@@ -9,7 +9,56 @@
 		<SynthesizeLinkMetadata>true</SynthesizeLinkMetadata>
 
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-	  	<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
+		<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
+
+		<DefaultLanguage>en</DefaultLanguage>
+
+		<!--#if (android)-->
+		<IsAndroid>false</IsAndroid>
+		<IsAndroid Condition="$(TargetFramework) != 'net6.0' AND $(TargetFramework) != 'netstandard2.1' AND $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">true</IsAndroid>
```

Comment:
Why do we need the tests against net6.0 and netstandard2.1? Isn't targetframework test sufficient?